test/integration/narrow/routes/endemics/dev-sign-in.test.js

```diff
@@ -62,5 +63,28 @@ describe('Dev sign in page test', () => {
       expect(res.statusCode).toBe(302)
       expect(res.headers.location).toEqual('/apply/endemics/check-details')
     })
+
+    test('POST to dev login successfully returns a 302 and redirects to journey', async () => {
```

Comment:
Could do with a proper title on this test

Reply:
Yeah good catch, updated with a better name